cherrypy/_cpnative_server.py

```diff
@@ -122,7 +123,7 @@ def __init__(self, server_adapter=cherrypy.server):
                        self.server_adapter.socket_file or
                        None)
 
-        wsgiserver.HTTPServer.__init__(
+        cheroot.server.HTTPServer.__init__(
```

Comment:
Probably super would be better, but I'll leave that as a change to follow. I'm aiming for minimal flux to adopt cheroot.